dirty.html

```diff
@@ -200,6 +200,7 @@
     </div>
     <div id="footer">
       <div>
-        <div id="span">Web Development Immersive Course Webpage</div> General Assembly London <div id="span">2013</div id="span">
```

Comment:
`id=span` seems not a very good naming... i think that bad developer was trying to create a `<span>`...

https://developer.mozilla.org/en/docs/Web/HTML/Element/span